.snippets/text/builders/manage/locate-sudo-account.md

```diff
@@ -0,0 +1,6 @@
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network){target=\_blank} underneath the **Properties** section.
```

Comment:
Sometimes we use sudo and sometimes we use appchain governor. I feel like using "appchain governor" was an afterthought and now there's a slight disconnect and sometimes we use them interchangeably but never state that they can be one in the same

Reply:
In my mind, when referring to the account, I think of sudo. When I think of privileged actions, It's the appchain governor that executes them.
Maybe we should start using something like "governor's account" instead of sudo?

Reply:
I don't like governor's because it is not that self-explanatory. For now, I would stick to SUDO. The main problem being that SUDO is not long-lived because in the future there might be decentralized chains? 

Unsure